### DISCUSSION TITLE
Warning in context with DOM and probs

### DISCUSSION THREAD

**Discussion by niklasrammelkamp** [neuefische/questions#480] [Web]
Hi there, I got a warning I don't really understand.

Here is the warning:
Warning: React does not recognize the `isActive` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isactive` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

I'm using the `isActive` state in my navbar component for each button. According if `isActive` is true or false the styling changes.
The Button that gets this state isn't a normal button, it's a Button component that changes to a button or next Link depending if it gets a `href` or not.
Something is wrong with using the probs in the Button component I guess but I don't really know where.
Please have a look at my code and let me know if you find s solution. 

Here is the summarized NavBar component:
```js
export default function Navbar() {

return (
<StyledNav>

        <Button variant="navButton" isActive={activeNav === "home"} href="/">
          {activeNav === "home" ? (
            <SVGIcon variant="home_filled" width="24" color="#23222" />
          ) : (
            <SVGIcon variant="home" width="24" color="#23222" />
          )}
        </Button>

        <Button
          variant="navButton" isActive={activeNav === "profile"} href="/profile"
        >
          {activeNav === "profile" ? (
            <SVGIcon variant="profile_filled" width="20" color="#23222" />
          ) : (
            <SVGIcon variant="profile" width="20" color="#23222" />
          )}
        </Button>

    </StyledNav>
);
}
```

And here is the Button component. 
shout-out to @doemser 

```js
export default function Button(props) {
  return (
    <StyledButton as={props.href && Link} {...props}>
      {props.children}
    </StyledButton>
  );
}
```

If you need more information, here is the [repo](https://github.com/niklasrammelkamp/capstone-project) or just ask for more.

**Comment by doemser** [comment]
Can you also provide your StyledButton component? There is neither a open PR nor a current branch for this in your repository.

**Reply by niklasrammelkamp** [reply]
Sure. The StyledButton isn't in an extra file. It's here:
https://github.com/niklasrammelkamp/capstone-project/blob/main/components/Button/index.js

**Comment by niklasrammelkamp** [comment]
I just recognized that the warning only comes at my localhost. It isn't shown in the vercel [preview](https://capstone-project-ruddy.vercel.app/).
